src/Groups/group_characters.jl

```diff
@@ -283,31 +283,31 @@ then `nothing` is returned.
 julia> Oscar.with_unicode() do
          show(stdout, MIME("text/plain"), character_table(symmetric_group(3)))
        end;
-Sym( [ 1 .. 3 ] )
+Permutation group of degree 3 and order 6
 
  2  1  1  .
  3  1  .  1
-           
+
    1a 2a 3a
 2P 1a 1a 3a
 3P 1a 2a 1a
-           
+
 χ₁  1 -1  1
 χ₂  2  . -1
 χ₃  1  1  1
 
 julia> Oscar.with_unicode() do
          show(stdout, MIME("text/plain"), character_table(symmetric_group(3), 2))
        end;
-Sym( [ 1 .. 3 ] ) mod 2
+Permutation group of degree 3 and order 6 mod 2
```

Comment:
Perhaps putting the `mod 2` in brackets would suffice to "disambiguate" the description. If not then we need a more explicit description such as `2-modular Brauer table of permutation group of degree 3`.

Reply:
I think I would "2-modular Brauer table of permutation group of degree 3"